etna/transforms/outliers.py

```diff
@@ -41,11 +62,12 @@ def fit(self, df: pd.DataFrame) -> "OutliersTransform":
 
         Returns
         -------
-        result: _OneSegmentTimeSeriesImputerTransform
-            instance with saved outliers
+        instance with saved outliers
```

Comment:
According to [Numpydoc style](https://numpydoc.readthedocs.io/en/latest/format.html#returns)(which we use as the Contribution guide suggests) you need to specify type for the return value